src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -5302,4 +5302,7 @@ nuget locals global-packages -list</value>
   <data name="LocalsCommandListDescription" xml:space="preserve">
     <value>List the selected local resources or cache location(s).</value>
   </data>
+  <data name="PackageCommandSuffixDescription" xml:space="preserve">
+    <value>Adds a suffix to the generated version</value>
```

Comment:
dot at the end of the sentence, and try to be more descriptive, this reads like the result version will be 1.1.0-something 123-SUFFIX